src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -45,17 +45,15 @@
 import static org.sonarqube.gradle.SonarUtils.isAndroidProject;
 
 /**
- * A plugin for analyzing projects with the <a href="http://redirect.sonarsource.com/doc/analyzing-with-sq-gradle.html">SonarQube Scanner</a>.
+ * A plugin for analyzing projects with the <a href="http://redirect.sonarsource.com/doc/analyzing-with-sq-gradle.html">SonarScanner for Gradle</a>.
  * When applied to a project, both the project itself and its subprojects will be analyzed (in a single run).
  */
 public class SonarQubePlugin implements Plugin<Project> {
```

Comment:
Can we rename the classname too?

Reply:
I think it only makes sense to rename if we actually rename the plugin id

Reply:
make sense